src/Microsoft.Kusto.ServiceLayer/HostLoader.cs

```diff
@@ -84,6 +85,9 @@ private static void InitializeRequestHandlersAndServices(ServiceHost serviceHost
             ConnectionService.Instance.InitializeService(serviceHost, dataSourceConnectionFactory, connectedBindingQueue, dataSourceFactory);
             serviceProvider.RegisterSingleService(ConnectionService.Instance);
 
+            ObjectExplorerService.Instance.InitializeService(serviceHost, connectedBindingQueue, WorkspaceService<SqlToolsSettings>.Instance, ConnectionService.Instance, serviceProvider);
+            serviceProvider.RegisterSingleService(ObjectExplorerService.Instance);
+
```

Comment:
This service was not initialized for Kusto. I wonder how that works. What is the reason to add it for Log Analytics?

Reply:
I added the initialize here instead of having the ObjectExplorerService being initialized through reflection. In this case, I'm trying to stay consistent with the other services.